### PR TITLE
Update output

### PR DESCRIPTION
**Pull request by daniel-butler-irl**
### Description

Updating the output to show the diff between the before and after json

**Check the relevant boxes:**
- [ ] Bug fix (nonbreaking change that fixes an issue)
- [x] New feature (nonbreaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [ ] CI related update (pipeline, etc.)

### Checklist

- [x] If relevant, a test for the change has been added or updated as part of this PR.
- [ ] If relevant, documentation for the change has been added or updated as part of this PR.

### Merge

- Merge using "Squash and merge".
- Make sure to use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents. The commit message determines whether a new version of the modules needs to be released, and if so, which semver number to use).

### CODE REVIEW

File: .secrets.baseline
Comment (ocofaigh) [inline .secrets.baseline:109]: @daniel-butler-irl Did you locally run against an old common-dev-assets version? This is reverting the version of DS in main branch

Comment (ocofaigh) [reply .secrets.baseline:109]: the pipeline is failing due to this

Comment (daniel-butler-irl) [reply .secrets.baseline:109]: Must have, although was certain I had updated it. Looking to resolve now

**Review by ocofaigh — APPROVED** [review]
LGTM
@toddgiguere want to give it a final review before merge?

**Review by toddgiguere — APPROVED** [review]

**Pull request merged**
